lesson_17/dags/sql/transfer_sales_data_lake_raw_to_bronze.sql

```diff
@@ -0,0 +1,29 @@
+DECLARE v_dag_logical_date DEFAULT CAST(CAST('{{ dag_run.logical_date }}' AS TIMESTAMP) AS DATE);
```

Comment:
I would keep this variable as its origin in Airflow - as TIMESTAMP. Because usually we need to compare it with logical_date when we debug a pipeline.